misk/src/main/kotlin/misk/CoordinatedService.kt

```diff
@@ -1,3 +1,5 @@
+@file:Suppress("DEPRECATION")
```

Comment:
Yuck. Errors as warnings mean we just suppress them which means we don’t even get to see them in IntelliJ. Yuck.

Reply:
Yeah I'm not happy about this, but it was the only way to get the code to compile without modifying the Gradle script 😦